icons/toml.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg">
+<path fill="#9c4221" d="M99.236 0v14.382h12.944v99.236H99.236V128H128V0z"/><path fill="#7E7F7F" d="M32.36 41.708v-15.82h63.28v15.82H71.91v69.034H56.09V41.708z"/><path fill="#9c4221" d="M28.764 0v14.382H15.82v99.236h12.944V128H0V0z"/>
+</svg>
```

Comment:
**Each** icon must have **spacing** on the sides, read https://github.com/PKief/vscode-material-icon-theme/blob/main/CONTRIBUTING.md#icon-spacing

I also changed the height of the icon to `24x24` px

```suggestion
<svg viewBox="0 0 24 24" width="24" height="24" xmlns="http://www.w3.org/2000/svg"><g><path fill="#9c4221" d="m 17.780906,1.5 v 2.3595469 h 2.123625 V 20.140453 H 17.780906 V 22.5 H 22.5 v -21 z"/><path fill="#7e7f7f" d="M 6.8090626,8.3427187 V 5.74725 H 17.190938 V 8.3427187 H 13.297734 V 19.668609 H 10.702266 V 8.3427187 Z" /><path fill="#9c4221" d="M 6.2190937,1.5 V 3.8595469 H 4.0954688 V 20.140453 H 6.2190937 V 22.5 H 1.5 v -21 z"/></g></svg>
```